src/aabb.js

```diff
@@ -16,6 +16,11 @@ export default class AABB {
         this._upperRight.add(vec);
     }
 
```

Comment:
We could also just add a `setPosition(vec)` method that sets the `lowerLeft` to `vec` and the `upperRight` to whatever that should be.